arithmetization/src/main/java/net/consensys/linea/corset/CorsetValidator.java

```diff
@@ -137,7 +137,7 @@ private List<String> buildCommandLine(Path traceFile, String zkEvmBin) {
     ArrayList<String> options = new ArrayList<>();
     // Determine options to use (either default or override)
     String flags =
-        System.getenv().getOrDefault("GOCORSET_FLAGS", "--report --report-context 2 --mir ");
+        System.getenv().getOrDefault("GOCORSET_FLAGS", "--report --report-context 2 --air ");
```

Comment:
Using `--air` is more accurate, but could have performance impacts when running on local machines.